### ISSUE TITLE
AgPasture ME

### ISSUE THREAD

**Issue by IrisVogeler** [APSIMInitiative/ApsimX#8003]
### What happened?

[AGPRyegrass].HarvestedME
only returns zeros

### What Operating System are you on?

Windows

**Comment by par456** [comment]
Hi, I just ran a check and I was able to have AGPRyegrass give me non-zero values in a test simulation, so there many a configuration problem with yours.

AGPRyegrass is an AgPasture crop, and it's HarvestedME is only updated when it has biomass removed, such as when the RemoveBiomass function is called from a script.

Now these crops are often used in grazing simulations, and grazing will not update the HarvestedME value, instead they track that in the GrazedME property (for example [SimpleGrazing].GrazedME).

Does that help with your situation? I can have a look at your apsimx file if required as well to figure out what has gone wrong.
